src/schema.js

```diff
@@ -180,29 +180,24 @@ class Schema {
   * @param {Object} schema The API definition.
   */
   generateDefaultApi(api, schema) {
-    Object.keys(schema).forEach((key) => {
-      const fnName = toCamelCase(key);
-      const outKey = schema[key].Out && schema[key].Out.length === 1 ? schema[key].Out[0].Name : -1;
-      const allowDelta = this.config.protocol.delta &&
-        IGNORE_DELTA_METHODS.indexOf(key) === -1 &&
+    Object.keys(schema).forEach((method) => {
```

Comment:
I couldn't help myself, restructured this code and added a bit more explanatory name to the generated function (shown in callstacks)

Reply:
Looks better IMHO! :) Especially nice to get rid of the `Object.defineProperty(...)`.

Reply:
Looks good! Especially nice to get rid of `Object.defineProperty(...)`.